src/app/foo/foo.component.html

```diff
@@ -10,7 +10,7 @@ <h5 class="card-title">tsteele pizza toppings</h5>
     <div>
       <button (click)='clearAll()'>Clear All</button>
       <button (click)='updateDisplayedToppings()'>Update Chosen Toppings</button>
-      Chosen toppings: {{ chosenToppings }}
+      Chosen 123 toppings: {{ chosenToppings }}
```

Comment:
This change shouldn't be here. We were just testing...

Reply:
I'll wait for you to fix up this PR so this change is not included before I accept the PR. Let me know if you would like help fixing it up. Thanks.